benchmarks/_benchmark.py

```diff
@@ -172,17 +172,24 @@ def record(
 
         return benchmark_result, output
 
-    def execute_command(self, command, capture_output=True):
+    def execute_command(self, command):
         try:
-            print(command)
-            result = subprocess.run(command, capture_output=capture_output, check=True)
+            print("voltrondata/labs-benchmarks child process:", command)
+            result = subprocess.run(command, capture_output=True, check=True)
         except subprocess.CalledProcessError as e:
-            print(e.stderr.decode("utf-8"))
```

Comment:
Previously this line would sometimes fail with an AttributeError: NoneType has no attribute "decode".